application/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "clean": "rm -R ./node_modules/ ./coverage/",
     "lint": "eslint -c .eslintrc \"./**/*.js\"",
-    "test": "NODE_ENV=test npm run test:unit && npm run test:integration",
+    "test": "NODE_ENV=test mocha ./tests/*.js",
```

Comment:
Why changing this line?

Reply:
Hehe, you got me :) 
Reason is:
a) tests run faster as mocha loads once, not twice
b) while developing, one can use the it.only or describe.only methods to let only one test or test group to run (because that's the one they're trying to fix). Without this change, it will run ALL unit tests if you use .only in integration, or ALL integration tests if you use .only in a unit test

Reply:
Thanks for the clearup! Sounds reasonable =)